src/codeflare_sdk/utils/generate_yaml.py

```diff
@@ -191,8 +192,11 @@ def get_default_kueue_name(namespace: str):
             namespace=namespace,
             plural="localqueues",
         )
-    except Exception as e:  # pragma: no cover
-        return _kube_api_error_handling(e)
+    except ApiException as e:  # pragma: no cover
+        if e.status == 404 or e.status == 403:
```

Comment:
What's there error code returned when the API is not there? is it 404 as well? I'm not sure about 403, WDYT?

Reply:
I'll test the "not found" possibility more thoroughly but as for the 403 forbidden error I came to that conclusion while testing the scenario where Kueue is set to `Removed` in the DSC.

The CRDs are not removed with the operator by design. What does change is the users permissions somehow. 

When trying to list the local queues in a namespace we are met with a 403 forbidden error.

Reply:
Possibly the 403 can be explained because the RoleBinding that grant read permission on local queues  is managed as part of the Kueue component, so when it's removed, that RoleBinding is deleted.

Another scenario that'd be good to test is when the CRD is not present. It won't be possible to do it with RHOAI, given Kueue is installed by default. Maybe just trying on a cluster with only KubeRay installed.

Reply:
Okay after giving this a go it seems 404 is correct when kueue does not exist on the cluster and 403 is acceptable when kueue has been removed via a DSC